ipn/localapi/localapi.go

```diff
@@ -1610,6 +1611,41 @@ func (h *Handler) serveTKAWrapPreauthKey(w http.ResponseWriter, r *http.Request)
 	w.Write([]byte(wrappedKey))
 }
 
+func (h *Handler) serveTKAVerifyDeeplink(w http.ResponseWriter, r *http.Request) {
```

Comment:
Is the plan for this to verify other deeplinks in the future? if not, maybe we should specialize the method name to be about the sign device deeplink?

Reply:
done